sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -117,6 +124,20 @@ export namespace ConnectionContext {
       audience: options.managementSessionAudience
     };
     connectionContext.managementSession = new ManagementClient(connectionContext, mOptions);
+    connectionContext.isConnectionClosing = function() {
```

Comment:
Can we use arrow functions to keep coding patterns consistent with definitions of other functions on the connection context? Same for `waitForConnectionReset`

Reply:
I intentionally did not want to use arrow functions for methods I added to the connectionContext so that they could be treated similarly as class methods (`this` inside the function refers to the connectionContext). Within `ConnectionContext.create` there weren't any methods being added to the connectionContext besides the ones I added as part of this PR.

The other method that exists on ConnectionContextBase, `refreshConnection`, also does not use an arrow function so that it can refer to itself using `this` instead of relying on a reference to itself via a closure.

Reply:
If it would make it clearer, I can do an Object.assign() to the connectionContext instead of assigning methods individually. That way the pattern doesn't look as similar to assigning an arrow function.

Reply:
It might be useful to comment it. 

I had a similar discussion (on my own PR) with @ramya-rao-a  and I think that it might look like it's not important and thus get auto-switched by some other programmer later.

Reply:
I did find a way to have TypeScript yell at you if you change this to an arrow function. I'll push the changes shortly :smile:

Reply:
This is now updated.